eng/pipelines/coreclr/superpmi-diffs.yml

```diff
@@ -21,6 +21,7 @@ pr:
     # so that this build can block dependency auto-updates (this build is currently ignored)
     include:
     - src/coreclr/jit/*
+    - src/coreclr/gcinfo/*
```

Comment:
What are these changes for?   (just curious)

Reply:
superpmi-asmdiffs was not running for this change, even though the gcinfoencoder is statically linked into the JIT and affects its compilation results. This change is ensuring that superpmi-asmdiffs runs even on gcinfoencoder changes (so that we can see [diffs](https://dev.azure.com/dnceng-public/public/_build/results?buildId=967694&view=ms.vss-build-web.run-extensions-tab)).